sdk/src/main/java/com/kin/ecosystem/history/view/OrderHistoryRecyclerAdapter.java

```diff
@@ -174,51 +159,50 @@ private boolean isFailed(Status status) {
 
         private String getActionText(Order item) {
             String actionText = "";
-            switch (item.getStatus()) {
-                case COMPLETED:
-                    actionText = TextUtils.isEmpty(item.getCallToAction()) ? "" : item.getCallToAction();
-                    break;
-                case FAILED:
-                    actionText = TRANSACTION_FAILED_MSG;
-                    if(item.getError() != null) {
-                        actionText = TextUtils.isEmpty(item.getError().getMessage()) ?
-                            TRANSACTION_FAILED_MSG : item.getError().getMessage();
-                    }
-                    break;
-                default:
-                    break;
+            if (item.getStatus() == Status.FAILED) {
```

Comment:
no completed text anymore?

Reply:
no, it was only for marketplace spend offers with coupon code.
And now we don't have it.